lib/ClusterShell/Worker/Rsh.py

```diff
@@ -122,7 +140,7 @@ class WorkerRsh(ExecWorker):
     Remote Copy (rcp) usage example:
        >>> worker = WorkerRsh(nodeset, handler=MyEventHandler(),
        ...                     source="/etc/my.conf",
-       ...                     dest="/etc/my.conf")
+       ...                     dest="/etc/my.conf.bak")
```

Comment:
why this change?

Reply:
Just noticed an error in the comment I believe. Did not think the source and destination should be the same. Had that change in the previous pull request.

Reply:
I don't think there is an error error. Rcp is for copying files from 1 host to another one. Let's imagine you want to copy `/etc/passwd` from `host1` to `host[2-10]` to have the same file everywhere? The source and destination will be the same file name (but not the same file).

Reply:
Fair point.

Reply:
Removed!